securedrop/i18n_tool.py

```diff
@@ -427,7 +450,8 @@ def commit_changes(
           commit: {current}
         """
         ).format(remote=args.url, name=name, authors=authors_as_str, code=code, current=current)
-        git("-C", args.root, "commit", "-m", message, *paths)
+        subprocess.run(["git", "commit", "-m", message, *paths], cwd=args.root)
+        log.debug(f"Committing with this message: {message}")
```

Comment:
Mostly just curious if there was a reason you added debug logging here.

Reply:
Sure! 
This assertion in the test suite
https://github.com/freedomofpress/securedrop/blob/3036ebbf9d0b49e2b3ad82851b0a38b30b8c2c43/securedrop/tests/test_i18n_tool.py#L312
checks that the stdout of the `git("-C", args.root, "commit", "-m", message, *paths)` command is present in a log (specifically, an object called caplog).

This confused me a bit, but I concluded that the git command I was replacing was piping the stdout to the log. I couldn't think of way to reproduce this behavior with subprocess, so I simply added a new log.debug statement.

Is this OK? Am I missing something?

Reply:
Ahh, thanks for the explanation. LGTM!